tests/test_diffraction_objects.py

```diff
@@ -319,8 +303,25 @@ def test_scale_to(org_do_args, target_do_args, scale_inputs, expected):
                 "d": None,
                 "offset": 0,
```

Comment:
this is a different issue, unrelated to what you are doing here, but I would rather have default offset to be None (set to zero if None).  I think this is more logical for my brain.  If I don't set an offset, the offset is None.